Makefile

```diff
@@ -210,53 +211,67 @@ test.integration: test.integration.dbless test.integration.postgres
 .PHONY: test.integration.enterprise
 test.integration.enterprise: test.integration.enterprise.postgres
 
-.PHONY: test.unit
-test.unit:
-	@go test -v -race \
+.PHONY: _test.unit
+_test.unit:
+	go test -v -race $(GOFLAGS) \
 		-covermode=atomic \
 		-coverpkg=$(PKG_LIST) \
 		-coverprofile=coverage.unit.out \
 		./internal/... \
-		./pkg/...
+		./pkg/... 2>/dev/null
```

Comment:
`2 >/dev/null` is present in `_test.unit`, so that the output of `stderr` would not be present in `test.unit`. Is this expected?

Reply:
Changed it to only have `2 >/dev/null` in prettified test output targets